### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

File: docs/commandline.rst [hunk 1]
```diff
@@ -90,11 +90,10 @@ each stencil are put in a subfolder named after the stencil:
 The line
 
 .. code-block:: python
-   :caption: my_stencils.gt.py
 
    # [GT] using-dsl: gtscript
 
-is equivalent to writing
+in ``my_stencils.gt.py`` is equivalent to writing
```
Comment (gronerl) [inline docs/commandline.rst:96]: Not really this PR, but do we want to propose the`.gt.py` ending?

File: docs/quickstart.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ We strongly recommended to create a virtual environment for any new project:
     pip install --upgrade pip wheel setuptools
 
 
-Then clone the GT4Py repository and install the local copy or install it directly from GitHub with `pip`.
+Then clone the GT4Py repository and install the local copy or install it directly from pypi.org.
```
Comment (gronerl) [inline docs/quickstart.rst:23]:
```suggestion
Then clone the GT4Py repository and install the local copy or install it directly from PyPI: `pip install gt4py`
```

Comment (gronerl) [reply docs/quickstart.rst:23]: Ok you say it below, but the section mentions a bunch of things and only shows one. maybe at least make it clear that pip==PyPI. I don't think all our users know ;)

File: setup.cfg [hunk 3]
```diff
@@ -106,7 +106,7 @@ testing =
 [options.entry_points]
 # CLI entry points to be installed
 console_scripts =
-    gtpyc = gt4py.cli:gtpyc
+    gtpyc = gt4py.cartesian.cli:gtpyc
```
Comment (gronerl) [inline setup.cfg:109]: We now use gt4pyc as shorthand for `gt4py.cartesian` in imports. I don't know a better naming but the names are very close.

**Review by gronerl — COMMENTED** [review]
Some points to discuss

File: setup.cfg (hunk 3 above)
Comment (havogt) [reply setup.cfg:109]: yeah, I agree, but wouldn't change now

File: docs/commandline.rst (hunk 1 above)
Comment (havogt) [reply docs/commandline.rst:96]: I don't know. Probably not.

**Review by gronerl — APPROVED** [review]
I'll open a separate PR removing the `.gt.py` ending from examples in docs.

**Comment by gronerl** [comment]
Re-ran seemingly spuriously failed tox test.

File: docs/commandline.rst (hunk 1 above)
Comment (egparedes) [reply docs/commandline.rst:96]: I think it's actually required by the CLI. I vaguely remember that Rico used some `importlib` features using the extension to implement the functionality

**Comment by egparedes** [comment]
bors try

File: docs/commandline.rst (hunk 1 above)
Comment (havogt) [reply docs/commandline.rst:96]: yes, I also tested it with a file that had just .py suffix.

**Pull request merged**